client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -99,18 +99,31 @@ const DataFilesToolbar = ({ scheme, api }) => {
     });
   };
 
+  const protectedFiles = ['.bash_profile', '.bashrc', '.bash_history'];
```

Comment:
Let's include `~/.ssh` as well. Anything else we can think of?

Reply:
authorized_keys maybe

Reply:
Here is what I could think of: 
- .APPDATA
- .Trash

Reply:
@jchuahtacc , what do you think of ☝️